### ISSUE TITLE
Opening video links via command line not working in 0.9.1 AppImage and Fedora RPM

### ISSUE THREAD

**Issue by n303p4** [FreeTubeApp/FreeTube#713]
**Behavior of program**
When a YouTube video link is submitted via the command line, the FreeTube window is raised to the top, but doesn't load the video.

**Expected behavior**
The video link should load the video in FreeTube.

**How to reproduce**
1. Launch the FreeTube AppImage.
2. Run `FreeTube-0.9.1.AppImage [any YouTube video URL]` from a terminal.
3. The FreeTube window is raised, but no video loads.

**Desktop 1**
- OS: Fedora
- Version: 32
- FreeTube: 0.9.1-beta

**Desktop 2**
- OS: Manjaro
- Version: 20.1.2
- FreeTube: 0.9.1-beta

**Other comments / alternative solutions / workaround**
Launching videos from the command line works fine in the 0.9.0 AppImage, which I am currently using.

**Comment by Synkky** [comment]
This is most likely **intended behavior**
According to the release notes of v0.9.0
> Adding a YouTube link as a parameter when opening FreeTube via the CLI will automatically open that link upon boot-up. **Only installed / non-portable versions are supported with this feature.**

And an AppImage is as portable as it can get.

**Comment by n303p4** [comment]
Tried this on the Fedora RPM and it doesn't work here, either.

**Comment by n303p4** [comment]
Still broken in the 0.9.2 Fedora RPM package on Fedora 33.

**Comment by dolandemort** [comment]
Can confirm this is also a problem for me. Fedora 32 and using the RPM package. Version `0.9.0` allowed me to open a youtube URL from the command line. The functionality stopped working thereafter.

**Comment by n303p4** [comment]
This is still broken as of 0.10.0. Any update on this yet?

**Comment by PrestonN** [comment]
I took a look into this. I never noticed the problem as I usually use the feature along with a browser extension.

The problem had to do with some slight changes with how the CLI detects commands to work with Mac. It was changed to only detect URLs that had the `freetube://` protocol at the beginning of the URL. This meant that anything sent without that would be ignored. If you append it to a URL with your current installation, you'll see the stated functionality.

I've updated it to accept any URL once again and it will be updated in the next release. Apologies for taking so long to get this done.
